src/components/DatePickerContainer/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+import { DateRangePicker, isInclusivelyBeforeDay } from 'react-dates' 
+import 'react-dates/lib/css/_datepicker.css'
+import './DatePickerContainer.css'
+import store from '../../store'
+import moment from 'moment'
```

Comment:
Let's sort this so that all library imports (like `moment`) happen before relative file imports (like `../../store`), and css imports happen last. Stylistic choice, but it will make things easier to scan for and find.